tools/epoch-sync/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "near-epoch-sync-tool"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+repository.workspace = true
+license.workspace = true
+publish = false
+
+[dependencies]
+anyhow.workspace = true
+clap.workspace = true
+tracing.workspace = true
+
+nearcore.workspace = true
+near-chain.workspace = true
+near-chain-configs.workspace = true
+near-epoch-manager.workspace = true
+near-primitives.workspace = true
+near-store.workspace = true
+
+[features]
+default = ["nearcore/new_epoch_sync"]
```

Comment:
If this is a required feature for this crate to work, I believe you want 

```toml
nearcore = { workspace = true, features = ["new_epoch_sync"] }
```

The rule of thumb to use in making the choice between these two options is: "would this crate compile if the default features were not enabled for some reason (e.g. somebody built code with the `--no-default-features` flag)." If the answer is "yes", then the current suggested approach is fine. Otherwise the feature has to be enabled unconditionally.